src/sentry/snuba/metrics/extraction.py

```diff
@@ -37,6 +37,10 @@
 
 logger = logging.getLogger(__name__)
 
+NEXT_SPEC_FEAT_FLAG = "organizations:on-demand-metrics-query-spec-version-two"
+# When an org starts using the new spec version, they will be able to use these on-demand functions
+NEXT_SPEC_VERSION_FUNCTIONS = ["count_unique"]
```

Comment:
Adding count_unique means this spec version time gets reset to the day we merge this, since `count_unique` will only be available from today or tomorrow, that means anything else we had in spec version 2 can't be used since flipping this early will break `count_unique`

Reply:
We agreed that this is okay.